freqtrade/persistence.py

```diff
@@ -404,6 +402,25 @@ def total_open_trades_stakes() -> float:
             .scalar()
         return total_open_stake_amount or 0
 
+    @staticmethod
+    def get_overall_performance() -> Dict:
+        pair_rates = Trade.session.query(
+            Trade.pair,
+            func.sum(Trade.close_profit).label('profit_sum'),
+            func.count(Trade.pair).label('count')
+        ).filter(Trade.is_open.is_(False))\
+            .group_by(Trade.pair) \
+            .order_by(desc('profit_sum')) \
+            .all()
+        return [
+            {
+                'pair': pair,
+                'profit': round(rate * 100, 2),
```

Comment:
should not this rounding be moved to the caller?
so that the function returned 'true' value of profit?

Reply:
you'll laugh - but i thought about the same too but somehow decided not to ...

Reply:
Yes, because it's better from the design point of view...

What if I would like to use it from the strategy? I'll need the 'pure' data.

Reply:
yes, absolutely - that's why i changed it ;)